adbe/adb_enhanced.py

```diff
@@ -432,16 +435,18 @@ def _get_top_activity_data():
         print_error_and_exit('Device returned no response, is it still connected?')
     for line in output.split('\n'):
         line = line.strip()
-        regex_result = re.search(r'ActivityRecord{.* (\S+)/(\S+)', line)
-        if regex_result is None:
-            continue
-        app_name, activity_name = regex_result.group(1), regex_result.group(2)
-        # If activity name is a short hand then complete it.
-        if activity_name.startswith('.'):
-            activity_name = '%s%s' %(app_name, activity_name)
-        return app_name, activity_name
+        if line.startswith('mFocusedApp'):
```

Comment:
Is this change intentional? I think this is the exact code that I removed in https://github.com/ashishb/adb-enhanced/commit/69bb243f9d4508279554d074ed93ddc8e01100c0

Reply:
Nope, I forked the master branch.

Reply:
Just do a `git pull origin master` in that case. Your branch has become stale. I changed this code <24 hours ago.